src/components/accordion-item/accordion-item.tsx

```diff
@@ -130,7 +130,12 @@ export class AccordionItem {
     const buttonId = `${guid}-button`;
 
     return (
-      <Host tabindex="0">
+      <Host
+        aria-expanded={toAriaBoolean(this.active)}
```

Comment:
> Is it possible to move these new attributes into the div inside the host so that they cannot be modified by an end user?

Unfortunately using the div wouldn't provide on-demand feedback as the user is expanding/collapsing the component. If we leverage the div, collapse/expand is read back, but only when navigating away and then back to the item.

Reply:
What if the tabindex is moved from the host to the div inside it?

Reply:
> What if the tabindex is moved from the host to the div inside it?

That does it for JAWS and NVDA. 🎉 Will commit it up to check on VoiceOver.